VisualPinball.Unity/VisualPinball.Unity/Extensions/MathExtensions.cs

```diff
@@ -24,6 +24,11 @@ public static Vector3 ToUnityVector3(this Vertex3D vertex)
 			return new Vector3(vertex.X, vertex.Y, vertex.Z);
 		}
 
+		public static Vertex3D FromUnityVector3(this Vector3 v)

```

Comment:
there's already a "ToVertex3D" which does the same, might have been added to mainline after you did your pass here

Reply:
Yup right I forgot to remove it after using the other one.